cloud/scope/network_load_balancer_reconciler.go

```diff
@@ -150,13 +155,19 @@ func (s *ClusterScope) CreateNLB(ctx context.Context, lb infrastructurev1beta2.L
 	}
 
 	backendSetDetails := make(map[string]networkloadbalancer.BackendSetDetails)
+	healthCheckUrl := lb.NLBSpec.BackendSetDetails.HealthChecker.UrlPath
+	if healthCheckUrl == nil {
+		healthCheckUrl = common.String("/healthz")
```

Comment:
Since we default this to `/healthz` should we call this out in the docs? Right now the comments only say
```
	// The path against which to run the health check.
	// Example: `/healthcheck`
```

for `isPreserverSourceIp` we say `// The value is false by default.`

Reply:
Added the default value statement